pages/settings.tsx

```diff
@@ -38,7 +38,7 @@ const Settings = () => {
               <h1 className="text-xs-center">{title}</h1>
               <SettingsForm />
               <hr />
-              <button className="btn btn-outline-danger" onClick={handleLogout}>
+              <button className="btn btn-outline-danger" data-cy="logout-btn" onClick={handleLogout}>
```

Comment:
The data-cy attribute is added correctly, but remember to update the corresponding Cypress test to use this attribute when selecting the logout button.